package.json

```diff
@@ -58,10 +58,8 @@
   },
   "devDependencies": {
     "prettier": "2.1.2",
-    "typescript": "^4.0.3",
-    "vue": "^3.0.0"
+    "typescript": "^4.0.5",
+    "vue": "^3.0.2"
   },
-  "peerDependencies": {
-    "vue": "^3.0.0"
-  }
+  "peerDependencies": {}
```

Comment:
I have updated `yarn.lock` and `typescript`
Could you please revert your changes in `package.json` and merge master into your branch?